spec/tictac_spec.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/logic'
+
+describe Game do
+  let(:new_game) { Game.new }
+  let(:player) { Player.new('Deyan', 'X') }
+  describe '#make_move' do
+    it 'check if it is correct' do
+      expect(new_game.make_move(player.team, 4)). to eql true
+    end
+    it 'Check if it is a string' do
+      expect(new_game.make_move(player.team, 'string')).to eql false
+    end
+    it 'check if the number is from 1 to 9' do
+      expect(new_game.make_move, 12).to eql false
+    end
+  end
+
+  describe '#check_if_win?' do
+    it 'Check if there is no winner' do
+      expec(new_game.check_if_win?).to eql false
+    end
+  end
+
+  describe '#check_if_move_done?' do
+    it 'Check if there is a move'
+    expect(new - game.check_if_win?(7)).to eql false
```

Comment:
- [ ] **[REQUIRED]** Missing `do - end` for it block. Please fix this.

- [ ] **[REQUIRED]** Please fix the `new - game` typo here
```ruby
Game#check_if_move_done? Check if there is a move
     Failure/Error: expect(new - game.check_if_win?(7)).to eql false
       `new` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
     # ./spec/tictac_spec.rb:26:in `block (3 levels) in <top (required)>'
```

- [ ] **[REQUIRED]** Even after we fix the `new - game` error, we still get a new error. Please fix it:

```ruby
Game#check_if_move_done? Check if there is a move
     Failure/Error:
       def check_if_win?
         winner = false
         win_sets = [[0, 1, 2], [0, 4, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [3, 4, 5], [6, 7, 8], [2, 4, 6]]
         win_sets.each do |p|
           winner = true if @board[p[0]] == @board[p[1]] && @board[p[0]] == @board[p[2]]
         end
         winner
       end
     
     ArgumentError:
       wrong number of arguments (given 1, expected 0)
     # ./lib/logic.rb:31:in `check_if_win?'
     # ./spec/tictac_spec.rb:26:in `block (3 levels) in <top (required)>'
```